src/hpc/traverse.c

```diff
@@ -204,17 +204,17 @@ static void TraverseRegionFrom(TraversalState * traversal,
 //   return CheckReadAccess(obj);
 // }
 
-static int IsSameRegion(TraversalState * traversal,Obj obj)
+static BOOL IsSameRegion(TraversalState * traversal, Obj obj)
 {
     return REGION(obj) == traversal->region;
 }
 
-static int IsMutable(TraversalState * traversal, Obj obj)
+static BOOL IsMutable(TraversalState * traversal, Obj obj)
 {
     return CheckReadAccess(obj) && IS_MUTABLE_OBJ(obj);
 }
 
-static int IsWritableOrImmutable(TraversalState * traversal, Obj obj)
+static BOOL IsWritableOrImmutable(TraversalState * traversal, Obj obj)
```

Comment:
Same as in a comment above: This function still returns `0` and `1`. Maybe I don't understand something, but it seems to me like this should be `FALSE` and `TRUE`?

Reply:
I'll change this particular function to return TRUE/FALSE now; but I never intended for this PR to be the final word on BOOL/TRUE/FALSE, but rather the beginning; in particular, there are lots of places that still use 0/1 not FALSE/TRUE. And I think that's fine overall (after all, comparisons like `foo < bar` also return 0 and 1), though it won't hurt to change it to TRUE/FALSE over time. But it's not necessary to do it all at once.

Reply:
Ok, fair enough. You're right, one could do that in future PRs.